src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -64,6 +64,13 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   if (!highlighted) {
     lines = markdownIt.utils.escapeHtml(str).split('\n');
   }
+
+  const startFrom = Number(token.attrGet('start-from'));
+
+  if (startFrom) {
+    // counter is incremented on each span, so we need to subtract 1
+    token.attrJoin('style', `counter-set: line ${startFrom - 1};`);
```

Comment:
```suggestion
    token.attrJoin('style', `counter-reset: line ${startFrom - 1};`);
```

[counter-reset](https://developer.mozilla.org/en-US/docs/Web/CSS/counter-reset) has better browser compatibility than [counter-set](https://developer.mozilla.org/en-US/docs/Web/CSS/counter-set)